src/factories/Axis.ts

```diff
@@ -214,5 +219,17 @@ module n3Charts.Factory {
 
       return axis;
     }
+
+    cloneAxis(): D3.Svg.Axis {
```

Comment:
We are in the `Axis` class anyway, maybe we should call this just `clone`. However, I also see the method above is called `updateAxisOrientation` and not `updateOrientation`. I would love to hear your comment on this!